### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by aleemont1**
Still didn't finish,  made it a little prettier and added some content.
The CI with gradle badge needs to be fixed but couldn't figure out how.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,14 +12,14 @@
     <!-- <img src="" alt="Logo" width="80" height="80"> -->
   </a>
 
-  <h3 align="center">HOMER</h3>
+  <h2 align="center">HOMER</h2>
 
   <p align="center">
     HOMER is an Open Source smart home emulator.
     <br />
```
Comment (michelececcacci) [inline README.md:19]: This whole part right here (the centered badges) feels a bit unnecessary, but LGTM

**Comment by michelececcacci** [comment]
Can you squash commits?

**Review by michelececcacci — APPROVED** [review]

**Comment by aleemont1** [comment]
> Can you squash commits?

Yep, that was my idea.

File: README.md (hunk 1 above)
Comment (aleemont1) [reply README.md:19]: > This whole part right here (the centered badges) feels a bit unnecessary, but LGTM

We can move them to the bottom or completely delete them, this is a template.

Comment (michelececcacci) [reply README.md:19]: i'd rather delete them, but that' s just my opinion.

Comment (aleemont1) [reply README.md:13]: Okie dokie. I'll remove them and then squash and merge ok?

Comment (michelececcacci) [reply README.md:13]: ok

**Pull request merged**